### PR TITLE
Set of small updates

### PR DESCRIPTION
**Pull request by akeranen**

### CODE REVIEW

File: draft-irtf-t2trg-rest-iot.md [hunk 1]
```diff
@@ -122,10 +124,12 @@ Since RESTful APIs are often simple and lightweight, they are a good fit for var
 The goal of this document is to give basic guidance for designing RESTful systems and APIs for IoT applications and give pointers for more information.
 
 Design of a good RESTful IoT system has naturally many commonalities with other Web systems.
-Compared to other systems, the key characteristics of many IoT systems include:
+Compared to other systems, the key characteristics of many RESTful IoT systems include:
 
 * need to accommodate for constrained devices, so with IoT, REST is not only used for scaling out (large number of clients on a web server), but also for scaling down (efficient server on constrained node)
 * data formats, interaction patterns, and other mechanisms that minimize, or preferably avoid, the need for human interaction
+* server endpoints are often highly constrained (e.g., simple sensor devices)
```
Comment (cabo) [inline draft-irtf-t2trg-rest-iot.md]: at least some of the server endpoints...

File: draft-irtf-t2trg-rest-iot.md [hunk 2]
```diff
@@ -367,18 +372,22 @@ Therefore applications should not rely on their order; see Section 3.3 of {{?RFC
 ## Representations
 
 Clients can retrieve the resource state from an origin server or manipulate resource state on the origin server by transferring resource representations.
-Resource representations have a media type that tells how the representation should be interpreted by identifying the representation format used.
+Resource representations have a media-type that tells how the representation should be interpreted by identifying the representation format used.
```
Comment (cabo) [inline draft-irtf-t2trg-rest-iot.md]: Is this a media-type or a content-type?

Comment (akeranen) [reply draft-irtf-t2trg-rest-iot.md]: I think both are OK/possible here, but I'd lean towards media-type for familiarity.

File: draft-irtf-t2trg-rest-iot.md (hunk 1 above)
Comment (akeranen) [reply draft-irtf-t2trg-rest-iot.md]: Yes. Should the text be tweaked? "some are often" would sound a bit weird :)

Comment (cabo) [reply draft-irtf-t2trg-rest-iot.md]: "Often, at least some of the servers..." would also work.

File: draft-irtf-t2trg-rest-iot.md (hunk 2 above)
Comment (cabo) [reply draft-irtf-t2trg-rest-iot.md]: content-type (media-type + optionally parameters)

maybe?

Comment (akeranen) [reply draft-irtf-t2trg-rest-iot.md]: SGTM. Fixed.

File: draft-irtf-t2trg-rest-iot.md (hunk 1 above)
Comment (akeranen) [reply draft-irtf-t2trg-rest-iot.md]: Maybe "often some classes of server endpoints (e.g., simple sensor devices) are highly constrained"?

**Comment by akeranen** [comment]
> Not a big fan of the "+", but WFM

Fixed and tweaked.

File: draft-irtf-t2trg-rest-iot.md (hunk 1 above)
Comment (akeranen) [reply draft-irtf-t2trg-rest-iot.md]: Latest incarnation:
> for some classes {{RFC7228}} of server endpoints, significant constraints, e.g., in energy consumption, and thus implementation complexity, may apply.

**Pull request merged**